Alloy/commands/compile/parsers/Ti.UI.Label.js

```diff
@@ -44,6 +44,10 @@ function parse(node, state, args) {
 		}
 	}
 
+	if ( node.getAttribute('verticalAlign') === 'center' ) {
+		node.setAttribute('verticalAlign', 'Titanium.UI.TEXT_VERTICAL_ALIGNMENT_CENTER');
```

Comment:
Similar change in behaviour comment

Reply:
> What's the desire for adding the `src` alias, is it purely because it's less characters than `image`? I'm not really a fan of adding an undocumented alias purely to avoid typing 2 characters.

I think src or source makes much more sense.  That would align us more with HTML, NativeScript, ReactNative, Xamarin, etc

Reply:
If we want to use `src` that change should come in the SDK imo